include/service/entities/article_service.js

```diff
@@ -132,11 +132,11 @@ module.exports = function ArticleServiceModule(pb) {
 
         //build out the ordering
         var order;
-        if (util.isArray(options.order)) {
+        if (util.isObject(options.order)) {
             order = options.order;
         }
         else {
-            order = [['publish_date', pb.DAO.DESC], ['created', pb.DAO.DESC]];
+            order = {'publish_date': pb.DAO.DESC, 'created': pb.DAO.DESC};
```

Comment:
I haven't gone back and looked at the mongo documentation yet but how can this guarantee order when hashes are are guaranteed to provide the hash keys back out in any order?

It was our experience early on that the mongo driver excepted two forms of order.  The array syntax as shown above and the object syntax which we used when there was only 1 field to sort on.  It was enormously confusing which is why we attempted to start moving everything to the array syntax.